assets/css/finder-hero.css

```
@@ -0,0 +1,75 @@
+/* Finder-window hero styles — loaded only by layouts/for-individuals/single.html
+   as a separate Tailwind sheet, so other pages do not ship the split-flap rules.
+
+   @reference imports Tailwind and the shared theme/utilities for @apply
+   resolution only — nothing from them is re-emitted here. Hugo pipes this file
+   to the Tailwind CLI with the project root as cwd and does not rewrite
+   @reference like it does @import, so the theme path is root-relative. The
+   @layer components wrapper keeps these rules in the same cascade layer they
+   would have inside main.css, so utility classes in the markup still win over
+   component rules. */
+@reference "tailwindcss";
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@reference`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)